Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -152,8 +152,9 @@ private extension Method {
             }
         }
         if !returnTypeName.isVoid && !isInitializer {
+            let mockNaming = MockNamingScheme.shared.createMockNaming(typeName: type.name)
```

Comment:
I think this should be possible without using a shared instance. `annotations` is globally available. So I suppose we could make an extension on `Annotations` to figure out the mock name
```swift
extension Annotations {
    func mockName(typeName: String) -> String {
        let prefix = mockPrefix ?? "Default"
        let suffix = mockSuffix ?? "Mock"
        return prefix + typeName + suffix
    }
}
```
Then we dont need a shared instance and it's available throughout. I suppose those global properties are a bit of an anti pattern, but since they're already provided by sourcery I think we should not worry about using them.

Reply:
Or an extension on `Type`
```swift
extension Type {
    var mockName: String {
        let prefix = mockPrefix ?? "Default"
        let suffix = mockSuffix ?? "Mock"
        return prefix + name + suffix
    }
}
```